toolchain-fitnesse-plugin/src/main/java/nl/praegus/fitnesse/responders/testHistory/RecentTestHistory.java

```diff
@@ -27,6 +29,14 @@ public RecentTestHistory(File historyDirectory) {
                 .collect(toList());
     }
 
+    public List<TestHistoryLine> getFilteredTestHistoryLines() {
+        return testHistoryLines.stream()
+                .filter(testHistoryLine -> !testHistoryLine.getPageName().contains("TearDown"))
```

Comment:
als ik nu een pagina heb met de naam BeetjeRaarTearDown dan wordt deze er onterecht uit gefilterd denk ik? Volgens mij moet je hier exact zijn. 

Verder denk ik dat je 1 regel met filter moet schrijven, als het een hele lange lijst is dan is het een beetje duur om 2 keer (en straks 4 keer) die hele lijst door te lopen. Ik weet niet of de compiler een optimalisatie doet met meerdere filters achter elkaar...